www/main/addons/ceremonies/components/CeremonyPane.jsx

```diff
@@ -25,10 +27,21 @@ const CeremonyPane = props => {
     applyTheme(theme);
   };
 
-  const toggleEnglishExplainations = isEnglishExplainations => {
-    store.setUserPref(`gurbani.ceremonies.ceremony-${token}-english`, isEnglishExplainations);
+  const toggleOptions = (toggleType, toggleVar) => {
+    store.setUserPref(`gurbani.ceremonies.ceremony-${token}-${toggleType}`, toggleVar);
     global.platform.updateSettings();
-    loadCeremonyGlobal(id);
+    const ceremonyToLoad = toggleType === 'rm' ? ceremoniesFilter.raagmalaMap[id] : id;
```

Comment:
there's a bug with having the toggle off and selecting  a different theme from the ceremony without toggling. The ceremony is loaded with raag mala when the toggle is off.